src/main/java/cs2103/v15_1j/jimjim/ui/TaskEventRowFactory.java

```diff
@@ -54,6 +54,11 @@ public int showAllDeadlineTaskAndEvents(LocalDate selectedDate){
 		int eventCounter = 0;
 		int deadlineTaskCounter = 0;
 		int addedCounter = 0;
+		boolean displayDate = true;
+		
+		if(selectedDate.equals(LocalDate.MIN)){
```

Comment:
this is still checking whether the parameter is equal LocalDate.MIN. Actually a better approach will be giving passing selectedDate as null. Then here you check if selectedDate is null, initialize it to LocalDate.MIN. I just don't like using equal LocalDate.MIN